### PR TITLE
Feature/m0078 ddg 291 replace classpath variables

### PR DESCRIPTION
**Pull request by frivieremicroej**

### CODE REVIEW

File: PlatformDeveloperGuide/coreEngine.rst [hunk 1]
```diff
@@ -347,17 +347,23 @@ application" mode is installed.
...
+Project. This MicroEJ module is always required in the build path of a MicroEJ project; 
+and all others libraries depend on it. This library provides a set of options.
+Refer to the chapter :ref:`application_options` which lists all available options.
 
-Another classpath variable named ``BON-1.2`` is available. This variable
-must be added to the build path of the MicroEJ Application project in
-order to access the :ref:`[BON] library <esr-specifications>`.
+::
 
+   <dependency org="ej.api" name="edc" rev="1.3.0"/>
```
Comment (frivieremicroej) [inline PlatformDeveloperGuide/coreEngine.rst:358]: Please refer to the latest version available in the Central Repository (everywhere)

File: PlatformDeveloperGuide/device.rst [hunk 2]
```diff
@@ -39,7 +39,13 @@ The properties file must / can contain the following properties:
...
 ===
 
-A classpath variable named ``DEVICE-1.0`` is available.
+The `Device API Module <https://repository.microej.com/artifacts/ej/api/device/>`_ 
+must be added to the :ref:`module.ivy <mmm_module_description>` of the MicroEJ 
+Application project in order to allow access to the Device library.
+
+::
+
+   <dependency org="ej.api" name="device" rev="1.+"/>
```
Comment (frivieremicroej) [inline PlatformDeveloperGuide/device.rst:48]: Please replace with latest version available in Central Repo (everywhere) (+ is invalid in MMM notation)

Comment (fpinel-microej) [reply PlatformDeveloperGuide/device.rst:48]: I did base this line on Central Repo https://repository.microej.com/artifacts/ej/api/device/1.0.2/
What would the correct line be then ?

File: PlatformDeveloperGuide/coreEngine.rst (hunk 1 above)
Comment (fpinel-microej) [reply PlatformDeveloperGuide/coreEngine.rst:358]: I think I did base this line on central repo as well, but now I see it is "@MMM_DEPENDENCY_DECLARATION@" in the README.md, is this a bug ? https://repository.microej.com/artifacts/ej/api/edc/1.3.0/

Comment (frivieremicroej) [reply PlatformDeveloperGuide/coreEngine.rst:358]: Let's wait for @gharelMicroEJ to publish the latest central repo with version 1.3.3

File: PlatformDeveloperGuide/device.rst (hunk 2 above)
Comment (frivieremicroej) [reply PlatformDeveloperGuide/device.rst:48]: 1.0.2

Comment (frivieremicroej) [reply PlatformDeveloperGuide/device.rst:48]: this API has not been updated for MMM - please also create an internal ticket

**Pull request merged**